odl/discr/grid.py

```diff
@@ -515,6 +515,40 @@ def insert(self, index, other):
                     self.coord_vectors[idx:])
         return TensorGrid(*new_vecs)
 
+    def remove(self, axes):
```

Comment:
Should be done using `__del__`, `remove` methods removes *by value*.

See
http://stackoverflow.com/questions/11520492/difference-between-del-remove-and-pop-on-lists